src/components/Footer.js

```diff
@@ -1,12 +1,15 @@
 import "./Footer.css";
 import { NavLink } from "react-router-dom";
+import { BsPlusSquare } from "react-icons/bs";
 
 export default function Footer() {
   return (
     <footer className="Footer">
       <nav className="Footer__navigation">
         <NavLink to="/">MyCakes</NavLink>
-        <NavLink to="/add">Plus</NavLink>
+        <NavLink to="/add">
+          <BsPlusSquare size="2rem" />
```

Comment:
I think inline style is not good practice. You could give it a class or `activeClassName` and style it in your css file.
I found this article on that topic (Method 3):
https://dev.to/ridhikgovind/how-to-style-your-react-router-links-using-styled-components-2350

Reply:
Thanks! Agree, will check for an alternative approach as I had issues with styling those elements and the icons in particular. :)